docs/tutorials/installation.md

```diff
@@ -1,10 +1,319 @@
 (installation)=
 
-# Installation
+# How to install JAX
 
-```{note}
-This is a placeholder for a section in the new {ref}`jax-tutorials`.
+This guide provides instructions for:
 
-For the time being, you may find some related content in the old documentation:
-- {doc}`../installation`
+- Installing JAX binary packages for supported platforms using `pip` or `conda`
+- Using [NVIDIA GPU Docker containers](#docker-containers-nvidia-gpu)
+- [Building JAX from source](developer.md#building-from-source)
```

Comment:
To ensure that we don't inadvertently break links, we should prefer cross-refrences here (i.e. ``` {ref}`label` ``` with the labeled section header specified via `(label)` in the line above the markdown heading).